### ISSUE TITLE
Regression: ${project} token malfunction

### ISSUE THREAD

**Issue by veelo** [SublimeLinter/SublimeLinter3#651]
Since 3.10.0, the `${project}` token does not expand to the project directory when used in user settings. 3.9.0 does work as advertised.

Instead, `${project}` expands to the path of the current file relative to the project directory.

Example:
project file: `C:\sarc\trunk4\sarc.sublime-project`:
```json
	"SublimeLinter":
	{
		"linters":
		{
			"epcomp": {
				"options": ["-i${project}\\pias\\source\\obj"]
			}
		}
	}
```
source file: `C:\sarc\trunk4\pias\source\complex.hdr`
Lint console:
```
SublimeLinter: epcomp: complex.hdr ['epcomp.exe', '-ipias\\source\\pias\\source\\obj']
```
This should have been (and under 3.9.0 is):
```
SublimeLinter: epcomp: complex.hdr ['epcomp.exe', '-iC:\\sarc\\trunk4\\pias\\source\\obj']
```

**Comment by veelo** [comment]
Maybe related/duplicate: #436.

**Comment by groteworld** [comment]
duplicate of #641, this should be fixed in v3.10.2
